index.html

```diff
@@ -0,0 +1,173 @@
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </div>
+          <div class="service-3">
+            <img src="assets/images/warehouse.png" alt="warehouse icon" />
+            <h3>Electronics</h3>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </div>
+          <div class="service-4">
+            <img src="assets/images/shopping.png" alt="shopping icon" />
+            <h3>Shoes</h3>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </div>
+          <div class="service-5">
+            <img src="assets/images/shopping(1).png" alt="shopping icon" />
+            <h3>T-shirt</h3>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </div>
+        </div>
+        <button class="see-whole-products" type="button">See The Whole Products</button>
+      </div>
+    </section>
+    <section class="featured-products">
+      <div class="container">
+        <h2 class="title">Featured products</h2>
+        <div id="featured-products"></div>
+        <div class="more-btn-container">
+          <span class="show-more-text">SHOW MORE</span>
+          <span class="arrow-down-up"><i class="fa fa-angle-down"></i></span>
+        </div>
+      </div>
+    </section>
+    
+    <section class="partners brandsLogo">
+      <div class="prodContainer">
+        <div class="container">
+          <h3 class="title">Our Partners</h3>
+          <div class="row">
+            <div class="col-5">
+              <img src="assets/images/logo-godrej.png" alt="godrej logo" />
+            </div>
+            <div class="col-5">
+              <img src="assets/images/logo-oppo.png" alt="oppo logo" />
+            </div>
+            <div class="col-5">
+              <img
+                src="assets/images/logo-coca-cola.png"
+                alt="coca-cola logo"
+              />
+            </div>
+            <div class="col-5">
+              <img src="assets/images/logo-paypal.png" alt="paypal logo" />
+            </div>
+            <div class="col-5">
+              <img src="assets/images/logo-philips.png" alt="philips logo" />
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <footer class="footer-home-mobile">
+      <div class="container">
+        <div class="foot-left">
+          <a href="index.html"><img src="assets/images/yaycommerceNobg.png" alt="logo" /></a>
+        </div>
+        <div class="foot-right">
+          <h4>
+            Copyright &copy; 2022 GeezTech Systems, Inc. All Rights Reserved
+          </h4>
+          <ul>
+            <li><a href="#" id="privacy">Privacy Policy</a></li>
+            <li><a href="#">Terms of Use</a></li>
+            <li><a href="#">Accessibility</a></li>
+            <li><a href="#">Sitemap</a></li>
+          </ul>
+        </div>
+      </div>
+    </footer>
+    <script src="assets/script/script.js"></script>
```

Comment:
In the mobile version, Our Partners and Footer section isn't showing up. Kindly fix this as well. 👍🏻 
<img width="303" alt="Screen Shot 2022-04-15 at 5 14 02 AM" src="https://user-images.githubusercontent.com/83048208/163496126-cf23d7c2-ac91-4c57-a743-471255780a9e.png">